charts/comet-site-v1/templates/service-account.yaml

```
@@ -0,0 +1,19 @@
+{{- if .Values.serviceAccount.create -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "comet-site.serviceAccountName" . }}
+  labels:
+    {{- include "comet-site.labels" . | nindent 4 }}
+  {{- with .Values.serviceAccount.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "comet-site.fullname" . }}-service-account
```

```suggestion
  name: {{ include "comet-site.fullname" . }}-service-account-token
```

might be better to understand

---

bd0ed60ddb32187ed934b368ae4261bbe9012492